cratedb_toolkit/io/iceberg.py

```
@@ -0,0 +1,231 @@
+
+    # Load from metadata file on AWS S3.
+    ctk load table \
+        "s3+iceberg://bucket1/demo/taxi-tiny/metadata/00003-dd9223cb-6d11-474b-8d09-3182d45862f4.metadata.json?s3.access-key-id=<your_access_key_id>&s3.secret-access-key=<your_secret_access_key>&s3.endpoint=<endpoint_url>&s3.region=<s3-region>" \
+        --cluster-url="crate://crate@localhost:4200/demo/taxi-tiny"
+
+    # Load from REST catalog on AWS S3.
+    ctk load table \
+        "s3+iceberg://bucket1/?catalog-uri=http://localhost:5001&catalog-token=foo&catalog=default&namespace=demo&table=taxi-tiny&s3.access-key-id=<your_access_key_id>&s3.secret-access-key=<your_secret_access_key>&s3.endpoint=<endpoint_url>&s3.region=<s3-region>" \
+        --cluster-url="crate://crate@localhost:4200/demo/taxi-tiny"
+
+    """  # noqa:E501
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (non-enabled: `E501`)

Remove unused `noqa` directive

(RUF100)